store/kv/kv_store.go

```diff
@@ -149,6 +159,12 @@ func (i *KeyValueIterator) Next(value *KeyValue) bool {
 			return false
 		}
 		value.Data = decoded
+		readBytes += int64(len(v.Value))
```

Comment:
is it a new PR? my previous comments are gone :)

Reply:
Yes, I addressed them in the new version.